modules/ROOT/pages/configure-studio-to-use-your-own-maven.adoc

```diff
@@ -0,0 +1,24 @@
+= Configure Studio to Use Your Own Maven
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio comes bundled with Maven but you can use Studio with an external Maven installation as well.
+
+. Navigate to Studio preferences:
+* In macOS, go to *Anypoint Studio > Settings…*.
```

Comment:
```suggestion
* In macOS, go to *Anypoint Studio > Settings*.
```

Reply:
Yep. That's our style.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-UIelements